### PR TITLE
Fix Scroll Theme for Dialogs

### PR DESCRIPTION
**Pull request by kvbhat**
Fix Scroll theme issue mentioned in https://github.com/theia-ide/theia/issues/2397

This patch fixes the 1st part (i.e. 'Open dialog has two scrollbars when files list is bigger than the window.')

Signed-off-by: Karthik Bhat <kv.bhat@samsung.com>

### CODE REVIEW

**Comment by akosyakov** [comment]
@kvbhat thank you for the contribution, we are in the process of moving to Eclipse and all contributors should sign [Eclipse Contributor Agreement](https://www.eclipse.org/legal/ECA.php). Let us know when you do it, that we can proceed.

**Comment by jbicker** [comment]
Hi @kvbhat,
thanks for the fix so far. 
I've tested it and saw that the styles of the scrollbars are different now. Please recheck that. Thanks!

Tested with Chrome Version 69.0.3497.100 (Official Build) (64-bit)

This is how it should look:
theia-scrollbar-good

This is how it looks now:
theia-scrollbar-bad

**Comment by kvbhat** [comment]
Hi @jbicker 
Thanks for the review and your time.
Fixed the same. I had missed a ',' in css file.
Please find the latest snapshots-

after-fix


For reference when we hover over the dialog previously it was getting shown as below-

dialog_before_fix

Thanks

**Comment by jbicker** [comment]
Hi, looks good to me now. Thanks!
Regarding the second part of #2397: as far as I can see it happens because of [this border](https://github.com/theia-ide/theia/blob/master/packages/filesystem/src/browser/style/file-dialog.css#L24). Since the `box-sizing` of the widget is set to `border-box`, the child (theia-TreeContainer, which height is set to 100%) shrunk by 2px. The next child (react-virtualized container) gets the height of the widget and so it is 2px higher, which causes the generation of the scrollbar.
Long story short: either we get rid of the border or we set the box-sizing [here](https://github.com/theia-ide/theia/blob/master/packages/filesystem/src/browser/style/file-dialog.css#L24) to `content-box`. 
I would say we do the first, because it still looks fine, we don't use such a border in other trees, and the second solution would slightly shift the layout. 
**cc**: @vitaliy-guliy

**Comment by kvbhat** [comment]
Hi @jbicker 
Thanks for the info. 
Yes removing the border indeed fixes the issue. I was not very sure previously why it was happening but now it is clear. Thanks for the input.

I modified the patch to remove border to prevent scrollbar being displayed when it is not required.
Please do share your input if this looks good to you.

Thanks

**Comment by kvbhat** [comment]
Thanks @jbicker , @akosyakov  for the review.
